src/functions/template-tags/ical.php

```diff
@@ -10,7 +10,10 @@
 	 * @return string URL for ical for single event.
 	 */
 	function tribe_get_single_ical_link() {
-		return tribe_get_ical_link();
+		$tec    = Tribe__Events__Main::instance();
+		$output = trailingslashit( $tec->getLink( 'single' ) ) . '?ical=1';
+
+		return apply_filters( 'tribe_get_single_ical_link', $output );
```

Comment:
Yes, it would probably be better. I will do it as soon as I get back. 

> On 15 Sep 2015, at 21:43, Matthew Batchelder <notifications@github.com> wrote:
> 
> I wonder if it might be better to modify Tribe__Events__iCal::get_ical_link() to accept a parameter (defaulted to home) to indicate which link it should return.
>